Purchases/Networking/Backend.swift

```diff
@@ -113,7 +113,12 @@ class Backend {
         }
 
         if let productInfo = productInfo {
-            body.merge(productInfo.asDictionary()) { _, new in new }
+            do {
+                body.merge(try productInfo.asDictionary()) { _, new in new }
```

Comment:
this can be simplified [using our extensions](https://github.com/RevenueCat/purchases-ios/blob/main/Purchases/FoundationExtensions/Dictionary+Extensions.swift#L86) as
```swift
body += try productInfo.asDictionary()
```

Reply:
Done thanks!